components/ActiveLink.js

```diff
@@ -0,0 +1,23 @@
+import {useRouter} from "next/router";
+
+const ActiveLink = ({href, children}) => {
+  const router = useRouter();
+  const handleClick = e => {
+    e.preventDefault();
+    router.push(href);
+  };
+
+  const isCurrentPath = router.pathname === href;
+
+  return (
+    <a
+      href={href}
+      onClick={handleClick}
+      style={{color: isCurrentPath ? "#735CDD" : "#8A878E"}}
+    >
+      {children}
+    </a>
+  );
+};
```

Comment:
In Next.js it is recommended to use the `Link` component instead of a standard `<a>` tag. The component will do clientside routing and handle the `onClick()` logic automatically.